resources/views/review.blade.php

```diff
@@ -1,3 +1,38 @@
+@php
+    $formattedDate = formatDate($review->created_at);
+    $backLabel = request('from') === 'movie' ? 'Back to movie' : (request('from') === 'profile' ? 'Back to profile' : 'Back');
+    $backHref = url()->previous();
+@endphp
+
 <x-layout>
-    <h1>Review</h1>
+    <x-section-header.back-link
+        title="{{ $review->movie->title }}"
+        extraLabel="Review"
+        href="{{ $backHref }}"
+        backLabel="{{ $backLabel }}"
+    />
+    <div class="relative mt-3 flex items-center gap-4">
+        <x-poster class="w-32 sm:w-40" src="{{ $review->movie->cover_image}}" />
+        <x-lucide-ellipsis-vertical
+            class="absolute top-0 right-0 size-6 cursor-pointer text-slate-50 sm:-top-10"
+        />
+
+        <div class="flex w-full flex-col gap-3">
+            <x-rating size="lg" rating="{{ $review->movie->rating_average }}" />
+
+            <div
+                class="flex items-center justify-between sm:flex-col sm:items-start sm:gap-1"
```

Comment:
```suggestion
                class="flex items-center justify-between sm:flex-col sm:items-start sm:gap-2"
```

thought it looked a bit strange so added a bit more padding in figma here